packages/client/src/runtime/core/engines/common/errors/engine-not-found/bundlerHasTamperedWithEngineCopy.ts

```diff
@@ -0,0 +1,17 @@
+import { weWouldAppreciateIfYouCouldShareInfo } from '../weWouldAppreciateIfYouCouldShareInfo'
+import { EngineNotFoundErrorInput } from './EngineNotFoundErrorInput'
+import { queryEngineCannotBeFound } from './queryEngineCannotBeFound'
+import { theseLocationsHaveBeenSearched } from './theseLocationsHaveBeenSearched'
+
+export function bundlerHasTamperedWithEngineCopy(input: EngineNotFoundErrorInput) {
+  const { queryEngineName } = input
+
+  return `${queryEngineCannotBeFound(input)}
+
+This is likely caused by a bundler that has not copied "${queryEngineName}" near the resulting bundle.
+Please try to make sure that "${queryEngineName}" is copied right near your bundle or "${input.expectedLocation}".
```

Comment:
I wonder whether "near" is the right term to use here. Maybe we should say "to the resulting bundle"?

Reply:
near feels a bit vague here

Reply:
@jkomyno I don't think "to the bundle" is the right wording either, as it would imply bundling the engine file as data inside the JS file.

"Next to the resulting bundle" maybe? I'm personally fine with "near" as well though.